### PR TITLE
Fixed a few IDE warnings and cleaned up the code

### PR DESCRIPTION
**Pull request by Mailaender**
https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1825 is performance related, but I doubt the impact will be huge. I just wanted to resolve the warnings in Visual Studio Code.

### CODE REVIEW

**Review by RoosterDragon — DISMISSED** [review]

File: OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs [hunk 1]
```diff
@@ -269,7 +269,6 @@ void RemoveCamera(Actor camera)
 
 			camera.QueueActivity(new Wait(info.CameraRemoveDelay));
 			camera.QueueActivity(new RemoveSelf());
-			camera = null;
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs:272]: Why did you remove this? It prevents `RemoveCamera` from queuing activities twice. Although I guess that shouldn't be a problem any more. If it is not a problem, is it possible to remove `beacon = null;` below too?

File: OpenRA.Mods.Common/Widgets/Logic/Ingame/CommandBarLogic.cs [hunk 2]
```diff
@@ -276,7 +276,7 @@ bool IsForceModifiersActive(Modifiers modifiers)
 			if (world.OrderGenerator is ForceModifiersOrderGenerator fmog && fmog.Modifiers.HasFlag(modifiers))
 				return true;
 
-			if (world.OrderGenerator is UnitOrderGenerator uog && Game.GetModifierKeys().HasFlag(modifiers))
+			if (world.OrderGenerator is UnitOrderGenerator && Game.GetModifierKeys().HasFlag(modifiers))
 				return true;
 
 			return false;
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Widgets/Logic/Ingame/CommandBarLogic.cs]:
```suggestion
			return world.OrderGenerator is UnitOrderGenerator && Game.GetModifierKeys().HasFlag(modifiers);
```
?

**Review by abcdefg30 — COMMENTED** [review]
Lgtm otherwise.

File: OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs (hunk 1 above)
Comment (RoosterDragon) [reply OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs:272]: `camera` here is a parameter, so assigning null to it wouldn't be visible outside this method anyway. Removing the `beacon = null` looks fine. The latter won't have been detected by the rules since it's inside a callback, but it looks safe to remove nonetheless.

File: OpenRA.Mods.Common/Widgets/Logic/Ingame/CommandBarLogic.cs (hunk 2 above)
Comment (Mailaender) [reply OpenRA.Mods.Common/Widgets/Logic/Ingame/CommandBarLogic.cs]: Okay.

File: OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs (hunk 1 above)
Comment (Mailaender) [reply OpenRA.Mods.Common/Traits/SupportPowers/ParatroopersPower.cs:272]: I also found another occurrence in `AirstrikePower` where I removed it as well.

**Review by RoosterDragon — APPROVED** [review]

**Review by abcdefg30 — APPROVED** [review]
